src/Tween.js

```diff
@@ -164,7 +164,7 @@ TWEEN.Tween.prototype = {
 
 	to: function (properties, duration) {
 
-		this._valuesEnd = Object.create(properties);
+		this._valuesEnd = JSON.parse(JSON.stringify(properties));
```

Comment:
This is only faster when using object literals, as in `Object.create({"foo": 123, "bar": 123})` vs `JSON.parse(JSON.stringify('{"foo": 123, "bar": 123}'))`.

Otherwise if we're dealing with references JSON.parse/stringify will be slower because there's nothing to parse if we're passing a reference to `Object.create`.

Performance test: https://jsperf.com/object-create-o-vs-json-parse-json-stringify-o/